models/cluster.go

```diff
@@ -151,7 +155,7 @@ type Cluster struct {
 	Kind *string `json:"kind"`
 
 	// The progress of log collection or empty if logs are not applicable
-	LogsInfo LogsState `json:"logs_info,omitempty" gorm:"type:varchar(2048)"`
+	LogsInfo LogsState `json:"logs_info,omitempty"`
```

Comment:
This will need a migration, right?

I think this is the same issue we ran into with ssh key.

I'm not reviewing all of the generated files, but we'll want to be sure we don't have any other changes like this (I just happened to open this one)

cc @filanov

Reply:
This is a swagger change that I missed